README.md

```diff
@@ -0,0 +1,11 @@
+# Docker
```

Comment:
We would like to avoid use of container because it is hard to maintain.  Container is good for deployment but it is cumbersome for source-level debugging.  For example, it's hard to compare the debug and optimized builds using a container.  It also adds complexity and overheads in instrumenting and profiling.

It's probably OK to add optional scripts for container, but we should not bless the use.

Reply:
Since container is not a blessed way to build the code, it should not be mentioned in README without a blessed approach is mentioned.

That is, at the moment, the project is not ready for a README.  This file should not exist (yet).

Reply:
Ok. I'll also move the README into `/contrib`

Reply:
In that way, how about adding the information as the comments in the docker file you add?